### ISSUE TITLE
Permission for inclusion in Cardinal

### ISSUE THREAD

**Issue by falkTX** [adbrant/ArableInstruments#21]
Hi there.
I would like to include your modules as part of [Cardinal](https://github.com/DISTRHO/Cardinal), which the code license allows but I am unsure regarding the graphics.

The README only states:

> The panel graphics in the res/ folder are copyright © Alex Brandt

But this is not a license, just a copyright notice. Would you mind placing a license for your artwork files? The [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) is the one that most closely matches GPL in spirit, there are also optional non-commercial and non-derivate clauses possible (though that restricts what can be done with the project). Otherwise CC-0/Public-Domain allows everything,

Thanks, have a great day!

**Comment by adbrant** [comment]
Hello!
I would be happy for these modules to be included in cardinal.
The artwork in the repo are modified versions of the originals from the AudibleInstruments repo, so I am not sure if I can change the licensing. I think you should be ok to use these in the same way as the originals.
I cant find the notice "The panel graphics in the res/ folder are copyright © Alex Brandt" in the repo, but I wouldn't have a problem removing it. 
Thanks,
Alex

**Comment by falkTX** [comment]
> The artwork in the repo are modified versions of the originals from the AudibleInstruments repo, so I am not sure if I can change the licensing. I think you should be ok to use these in the same way as the originals.

Fair enough. Luckily Emilie already gave thumbs up for its use in Cardinal.
You give permission too, so all should be fine.

> I cant find the notice "The panel graphics in the res/ folder are copyright © Alex Brandt" in the repo, but I wouldn't have a problem removing it.

It was added in 224610daf5e8053c31e3fc55acf7c8187314fe6b for the v2 branch.

**Comment by falkTX** [comment]
FYI I have tweaked the design a bit, because to be honest I find the regular one a bit ugly, and too bright.
So I inverted the colors, made the panel outline work (as any other modules have) and also cleaned up and made the background png blurry.
This is the result:

![image](https://user-images.githubusercontent.com/1334853/156007801-f47a0560-186f-4e3c-999a-ae268a40d44a.png)

I would like to use this version for Cardinal, is that ok with you?
If you like the design I can submit a PR, otherwise it will be specific to Cardinal. Due to me changing the files we cannot have both yours and my modified version at the same time though..

**Comment by falkTX** [comment]
Did similar things for Neil now too.

![Screenshot_20220228_154502](https://user-images.githubusercontent.com/1334853/156013013-42d4953c-8f92-4744-b368-3b577b8b4db2.png)

**Comment by falkTX** [comment]
Hi.

I adjusted the panels to keep in mind the original artwork. Should have realized earlier that there was a point for this specific artwork and modules names..
So "Joni" one now keeps the original painting in there, in a similar fashion to what "Neil" has.

![image](https://user-images.githubusercontent.com/1334853/190447178-4c1e6e47-a336-4f89-b3b4-c6d8cf7f45ff.png)

Should I submit these changes as pull request? Would you be interested on them?

**Comment by adbrant** [comment]
Sorry about the delay, yes I am interested in these changes!
Thanks,
Alex
